added series inductance capacitor to pf to pp converter

mfisch42 opened the pull request.

rbolgaryn commented:
also please check if the documentation is up to date (especially the list of supported elements):

https://github.com/e2nIEE/pandapower/blob/develop/doc/converter/powerfactory.rst

mfisch42 commented:
@rbolgaryn I added a changelog entry and a entry in the documentation. Further I updated the fcn as you mentioned ... 
could you double check it again, pls? :) thanks in advance!
